extensions-core/parquet-extensions/pom.xml

```diff
@@ -209,10 +205,6 @@
           <groupId>org.apache.yetus</groupId>
           <artifactId>audience-annotations</artifactId>
         </exclusion>
-        <exclusion>
-          <groupId>org.apache.avro</groupId>
-          <artifactId>avro</artifactId>
-        </exclusion>
```

Comment:
See lines 83 - 86 there is another exclusion for this library. I'm not sure if that exclusion should be removed as well.

Reply:
It can be removed as well, the top-level explicit include would override the avro versions that these deps try to pull in